drivers/espi/Kconfig.xec

```diff
@@ -37,4 +37,19 @@ config ESPI_PERIPHERAL_UART_SOC_MAPPING
 	  This tells the driver to which SoC UART to direct the UART traffic
 	  send over eSPI from host.
 
+config ESPI_SAF
+	bool "XEC Microchip ESPI SAF driver"
+	depends on ESPI_FLASH_CHANNEL
+	default y
```

Comment:
Do not enable by default, some board designs do not support SAF

Reply:
Change to "n" in next rebase